tensorboard/webapp/runs/views/runs_table/runs_data_table.ts

```diff
@@ -89,15 +89,15 @@ export class RunsDataTable {
   }
 
   getRunIds() {
-    return this.data.map((row) => row.id);
+    return (this.data || []).map((row) => row.id);
```

Comment:
I also find this a bit difficult to understand. this.data is a required property typed as TableData[] so the `|| []` part of `this.data || []` should never be evaluated in practice.

Are you adding this just because of some deficiency in the test? If so, it would be better to fix the test.